src/hera/_cli/base.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from pathlib import Path
+
+from cappa import Arg, Subcommands, command
+from typing_extensions import Annotated
+
+
+@dataclass
+class Hera:
+    subcommand: Subcommands[Generate]
+
+
+@command(help="Subcommands for converting to/from hera Workflows.")
```

Comment:
We have more generic intentions (see #670 examples) for this subcommand:

```suggestion
@command(help="Subcommands for generating yaml, code, and docs from Hera Workflows.")
```